wagtail/admin/templatetags/util.py

```diff
@@ -0,0 +1,124 @@
+from inspect import getfullargspec, unwrap
+
+from django.core.exceptions import ImproperlyConfigured
+from django.template.library import InclusionNode, parse_bits
+
+
+class PartialNode(InclusionNode):
+    """
+    Splits an existing template into to based on a string provided in the template
+    to act as two separate virtual (partial) templates for a start & end either
+    side of the split string.
```

Comment:
Suggested change: " Splits an existing template into two based on a string provided in the template.
    They will act as two separate virtual (partial) templates for a start and end which sit either
    side of the split string."